### PR TITLE
Let numbers with commas can also be aligned to right

### PR DESCRIPTION
**Pull request by WakeupTsai**
From:
|---- 530 | --- 12,960 |
To:
|---- 530 | 12,960 --- |

### CODE REVIEW

**Comment by mattn** [comment]
Could you please add test?

**Comment by WakeupTsai** [comment]
@mattn Hi, I add the test, please check :)

**Comment by mattn** [comment]
Thanks.

File: table.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ const (
 )
 
 var (
-	decimal = regexp.MustCompile(`^-*\d*\.?\d*$`)
+	decimal = regexp.MustCompile(`^(?:\d{1,3}(?:,\d{3})*|\d+)(?:\.\d+)?$`)
```
Comment (mattn) [inline table.go]: `^-?(?:\d{1,3}(?:,\d{3})*|\d+)(?:\.\d+)?$`

File: table.go [hunk 2]
```diff
@@ -36,7 +36,7 @@ const (
...
 	percent = regexp.MustCompile(`^-*\d*\.?\d*$%$`)
```
Comment (mattn) [inline table.go]: please fix this too.  `^-?\d*\.?\d*$%$`

**Comment by mattn** [comment]
Thanks. LGTM.

**Pull request merged**